gce/survival-training/README-tf-estimator.md

```diff
@@ -0,0 +1,404 @@
+
+We will have to install `pip`, though:
+
+```bash
+sudo apt-get install python-pip
+```
+
+And now let us install `tensorflow-gpu`:
+
+```bash
+pip install tensorflow-gpu
+```
+
+You can check that tensorflow has access to your GPUs by starting up your Python interpreter and creating a TensorFlow session:
+
+```python
+>>> import tensorflow as tf
+>>> tf.Session()
+```
+
+If this errors out, there is a problem (please raise an issue in this repo).
+
+
+#### Cleaning up your home directory
+
+Now that we've got everything set up, let us clear up our home directory a bit:
+
+```bash
+sudo rm -r cuda/
+mkdir image-setup
+mv cud* image-setup/
+```
+
+
+### Creating the image
+
+And now, we can create the image. It is possible to create an image from a running instance, but it is much safer to do so from a stopped instance, so let us take the safe route. Stop the `cifar10-estimator` instance either in the Cloud Console or using `gcloud` as follows:
+
+```bash
+gcloud compute instances stop cifar10-estimator
+```
+
+You can easily create the image in the Cloud Console from the disk attached to the VM we just stopped by clicking on the "Create Image" button at the top of the [Images page](https://console.cloud.google.com/compute/images) and creating an image from the disk attached to the VM instance in question (which should have the same name). Here, let me show you how to create an image from the command line:
+
+```bash
+gcloud compute images create gpu-tensorflow --source-disk cifar10-estimator --source-disk-zone us-west1-b
+```
+
+This concludes the image creation process. It *was* fairly involved, but the beauty of this is that you only have to do it once. Although no one can offer such an image publicly (because of the step where you had to register as an nvidia developer), once you have created your base image, you barely need to think about this process at all. [Images are shareable between GCP projects](https://cloud.google.com/compute/docs/images/sharing-images-across-projects), for example, so you can just pass it along from project to project as necessary. In short, this is worth doing (exactly) once.
+
+In the case of this example, we can just use the same instance from which we created the image to train a classifier on the CIFAR-10 dataset. If you wanted to do something like simultaneously train with different sets of hyperparameters, though, you would create a new VM based off of the same image, and create a separate training job on that with the new hyperparameter specification.
+
+
+## Trainer CLI
+
+Fortunately, [the CIFAR-10 estimator](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) example [comes wrapped in a CLI whose semantics are very similar to those of our dummy trainer](https://github.com/tensorflow/models/blob/master/tutorials/image/cifar10_estimator/cifar10_main.py). That is the interface we will use for our training job.
+
+
+### Arguments
```

Comment:
Ditto my previous response about devsite going into insufficient detail (in my opinion).

Reply:
in this case I was referring to the details about the training script (the #Arguments) section.

Reply:
Ah okay, I misunderstood. I initially didn't have this paragraph, but then introduced it because the whole thing felt too detached from the actual application. I am okay either way.

@dizcology : If you feel this is distracting, I am okay taking it out. @amygdala , @puneith , @thedriftofwords : Any opinions?